flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/options/SourceOptions.java

```diff
@@ -111,4 +112,15 @@ public class SourceOptions {
                                     + " The table chunks would use evenly calculation optimization when the data distribution is even,"
                                     + " and the query for splitting would happen when it is uneven."
                                     + " The distribution factor could be calculated by (MAX(id) - MIN(id) + 1) / rowCount.");
+
+    @Experimental
+    public static final ConfigOption<Boolean> SCAN_INCREMENTAL_CLOSE_IDLE_READER_ENABLED =
+            ConfigOptions.key("scan.incremental.close-idle-reader.enabled")
+                    .booleanType()
+                    .defaultValue(false)
```

Comment:
I think this option's default value should be `true`  , for the old flink version which don't support should be setted `false` by user.

Reply:
@Jiabao-Sun As you said , flink disabled by default. So ignore this comment. :)